content/en/docs/started/workstation/getting-started-minikf.md

```diff
@@ -3,17 +3,8 @@ title = "MiniKF"
 description = "A fast and easy way to deploy Kubeflow on your laptop"
 weight = 35
                     
-+++
```

Comment:
This is needed to deploy correctly the page..I've learned this from my last PR. :)